src/renderer/components/UnblockContacts.js

```diff
@@ -24,7 +24,7 @@ class UnblockContacts extends React.Component {
     const tx = window.translate
     confirmation(tx('ask_unblock_contact'), yes => {
       if (yes) {
-        ipcRenderer.sendSync('dispatchSync', 'unblockContact', contact.id)
+        ipcRenderer.send('unblockContact', contact.id)
```

Comment:
Apparently, `unblockContact` was called both sync and async from two different places. This behavior has now changed since from now on always call it async.